src/sage/functions/bessel.py

```diff
@@ -294,8 +294,8 @@ class Function_Bessel_J(BuiltinFunction):
         sage: f.integrate(x)
         1/24*x^3*hypergeometric((3/2,), (5/2, 3), -1/4*x^2)
         sage: m = maxima(bessel_J(2, x))
-        sage: m.integrate(x)
```

Comment:
I think this is a pretty useless test. What does anyone care about the maxima interface here?

In any case, `f.integrate()` _already_ uses maxima by default. We could specify `f.integrate(x,algorithm='maxima')` to ensure that the maxima interface is still tested in case the default ever changes... but again, who wants to read about the maxima interface here?